app/services/ClaimService.scala

```diff
@@ -42,8 +42,8 @@ class ClaimService @Inject() (
     maybeRegistrationRequest match {
       case Some(value) =>
         connector.submitClaim(value, correlationId(hc)).map { clientClaimResponse =>
-          if (clientClaimResponse.result.map(_.caseId).nonEmpty)
-            clientClaimResponse.result.map(_.caseId).get
+          if (clientClaimResponse.caseId.nonEmpty)
+            clientClaimResponse.caseId.get
```

Comment:
I know this was already like this, but can this be refactored to get rid of the "if" and "get" statements?

Reply:
Yes, Of course. I wanted you to see it exactly as it was, then you could decide whether you preferred to refactor slightly or go for minimal change. I will do it now.